lists/index.js

```diff
@@ -1,9 +1,11 @@
 const Stuff = require('./stuff')
 const FAQ = require('./faq')
+const projectDetails = require('./projectDetail')
```

Comment:
Maybe change projectDetails to ProjectDetails for better consistency with the rest of the lists? :)

The same in line 8.

Reply:
Done @Vogric